composer.json

```diff
@@ -13,20 +13,38 @@
             "email": "stepan.mocjak@guava.cz"
         }
     ],
-    "require": {},
+    "scripts": {
+        "phpstan": "vendor/bin/phpstan analyze"
+    },
     "require-dev": {
-        "orchestra/testbench": "^8.0"
+        "orchestra/testbench": "^8.0",
+        "phpstan/phpstan": "^1.11",
+        "larastan/larastan": "^2.9",
+        "spatie/phpunit-watcher": "^1.24",
+        "laravel/pint": "^1.16"
     },
     "autoload": {
         "psr-4": {
             "Guava\\LaravelPopulator\\": "src/"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/",
+            "Guava\\LaravelPopulator\\": "src/",
+            "Guava\\LaravelPopulator\\Database\\Factories\\": "database/factories"
+        }
+    },
     "extra": {
         "laravel": {
             "providers": [
                 "Guava\\LaravelPopulator\\PopulatorServiceProvider"
             ]
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "pestphp/pest-plugin": true
+        }
     }
```

Comment:
Since you've decided to use PHPUnit instead, can we delete the Pest plugin here?